atlas/osutils.nim

```diff
@@ -0,0 +1,59 @@
+## OS utilities like 'withDir'.
+## (c) 2021 Andreas Rumpf
+
+import os, strutils, osproc
+
+template withDir*(dir, body) =
```

Comment:
This is literally on Fusion 
https://nim-lang.github.io/fusion/src/fusion/scripting.html#withDir.t%2Cstring%2Cuntyped

Reply:
Cool, but this way I don't depend on Fusion. I also might want to change the implementation in a flexible way (adding debug output etc).